src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -39,9 +39,11 @@ public static function getExtendedTypes(): iterable
         ];
     }
 
-    public function finishView(FormView $view, FormInterface $form, array $options)
+    public function finishView(FormView $view, FormInterface $form, array $options): void
     {
         if (!$options['autocomplete']) {
+            $view->vars['autocomplete'] = false;
```

Comment:
Afterwards I wonder if that's a good idea... Might be better to leave it as it instead :thinking:

Reply:
That's a good question! I think we keep it. If you install ux-autocomplete, then you can know the variable is always available for text/choice types. But, to avoid collisions and for clarity, let's:

```suggestion
            $view->vars['uses_autocomplete'] = false;
```

Reply:
I totally agree, that's why I used `is_autocomplete` in my first message in issue :+1: